testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/client/RedirectTest.java

```diff
@@ -36,6 +39,12 @@
    {
       WebArchive war = TestUtil.prepareArchive(RedirectTest.class.getSimpleName());
       war.addClasses(PortProviderUtil.class);
+      war.addAsManifestResource(PermissionUtil.createPermissionsXmlAsset(
+              new PropertyPermission("ipv6", "read"),
+              new RuntimePermission("getenv.RESTEASY_PORT"),
```

Comment:
Same concerns as with "org.jboss.resteasy.port" property permission

Reply:
The test calls PortProviderUtil in arquillian-utils.  This class is calling 
System.getenv and/or System.getProperty however putting these calls in a privileged 
block is of no help because the test itself is not running in a security manager.